test/test_helper.rb

```diff
@@ -12,7 +12,7 @@
 require 'minitest/reporters'
 require_relative '../db/seminaries'
 require 'vcr'
-require "mocha/mini_test"
+require 'mocha/minitest'
```

Comment:
wow how did it even work before 🤔 scary.

Reply:
This was a deprecation, for now both will work, but in the future the underscore will stop working 🙂